generator/src/main/java/com/scottlogic/deg/generator/outputs/datasetwriters/JsonFormatter.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.generator.outputs.datasetwriters;
+
+import com.scottlogic.deg.generator.outputs.GeneratedObject;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+
+public class JsonFormatter implements RowOutputFormatter<String> {
+
+    @Override
+    public String format(GeneratedObject row) {
+        ArrayList<String> strValue = new ArrayList<>();
```

Comment:
I agree with @SL-Mark; use Jackson to serialise the 'row' as JSON and return this.